datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/MeResolver.java

```diff
@@ -60,6 +60,7 @@ public CompletableFuture<AuthenticatedUser> get(DataFetchingEnvironment environm
         platformPrivileges.setManageDomains(canManageDomains(context));
         platformPrivileges.setManageIngestion(canManageIngestion(context));
         platformPrivileges.setManageSecrets(canManageSecrets(context));
+        platformPrivileges.setManageTokens(canManageTokens(context));
```

Comment:
Why are we not calling this "manageAllAccessTokens"? We try very hard to keep things consistent to reduce the confusion of an already complex thing. 

Can we do that here?

Reply:
So it matters here but not in policies land?

Reply:
I see you've updated that. Nice